source/plugin/internals/mixins.rst

```diff
@@ -0,0 +1,117 @@
+=============
+Plugin Mixins
+=============
+
+`Mixins <https://github.com/SpongePowered/Mixin>`_ can be used to modify classes at runtime before they are loaded. You
+can use them in plugins if you want to optimize a part of the game specifically for your server - without having to fork
+Sponge. The modifications will be bundled directly with your plugin and are only active as long as the plugin is loaded.
+
+.. seealso::
+
+    `Mixin documentation <https://github.com/SpongePowered/Mixin/wiki>`_
+        Mixin documentation including an introduction to Mixins.
+    `Example plugin <https://github.com/SpongePowered/Cookbook>`_
```

Comment:
Which"Example Plugin"? BTW, the cookbook recipes are largely out of date (API 3.0.0)